tools/webtransport/h3/handler.py

```diff
@@ -61,3 +61,15 @@ def session_closed(session: WebTransportSession,
                      (by a CLOSE_CONNECTION QUIC frame for example).
     """
     pass
+
+def stream_reset(session: WebTransportSession,
```

Comment:
Could you also update Handlers section in tools/webtransport/README.md, maybe in a follow-up PR?

Reply:
Done.